debian/ubuntu-advantage-tools.postinst

```diff
@@ -265,6 +265,24 @@ mark_reboot_cmds_as_needed() {
     add_notice "$msg_name"
 }
 
+check_wrong_fips_metapackage_on_cloud() {
+      fips_metapkg="ubuntu-fips"
+
+      cloud_id=""
+      if which "cloud-id" > /dev/null ; then
```

Comment:
Let's run `shellcheck -x debian/<script>` everytime we touch debian/* script files to ensure we don't erode our scripts unintentionally.

Minor nit raised here  by shellcheck:
      if which "cloud-id" > /dev/null ; then
         ^---^ SC2230: which is non-standard. Use builtin 'command -v' instead.

```suggestion
    if command -v cloud-id > /dev/null 2>&1; then
        cloud_id=$(cloud-id)
    fi
```